entry_tasks/46_calculator/main.py

```diff
@@ -0,0 +1,18 @@
+if __name__ == "__main__":
+    a = int(input('Enter first number: '))
+    b = int(input('Enter second number: '))
+    operation = input('''
+    Enter operator:
+    + for "addition"
+    > for "greater than"
+    or another operator
+    ''')
+
+if operation == '+':
+    print('{} + {} is '.format(a, b))
+    print(a + b)
+elif operation == '>':
+    print('{} > {} is '.format(a, b))
+    print(a > b)
+else:
```

Comment:
not very powerful calculator :D but don't wont to bother you since you got the original idea